src/main/scala/slamdata/SbtSlamData.scala

```diff
@@ -273,6 +287,70 @@ object SbtSlamData extends AutoPlugin {
   private def transferScripts(baseDir: File, srcs: String*) =
     srcs.foreach(src => transfer(src, baseDir / "scripts" / src, Set(OWNER_EXECUTE)))
 
+  private def slamdataVersionCheck = Def.task {
+    import scala.sys.process._
+
+    val logger: Logger = streams.value.log
+    def relog(line: String): Unit = {
+      val level = """\[([^]]*)\].*""".r
+      line match {
+        case level("debug") => logger.debug(line)
+        case level("info") => logger.info(line)
+        case level("warn") => logger.warn(line)
+        case level("error") => logger.error(line)
+        case level("success") => logger.success(line)
+        case _ => logger.info(line)
+      }
+    }
+    val logToSbt = ProcessLogger(relog _)
+
+    val getCsbt = "wget -O csbt https://github.com/coursier/sbt-extras/raw/master/sbt"
+    val csbtUpdate: Seq[String] = Seq(
+      "/usr/bin/env",
+      "bash",
+      "./csbt",
+      """set versionReconciliation += "com.slamdata" % "*" % "strict"""",
+      "update")
+    val sbtUpdate = "./sbt update"
+
+    logger.info("running slamdata version check")
+
+    val pluginFile = scala.reflect.io.File("project/coursier.sbt")
```

Comment:
I would rather use `java.io.File` here directly, since `scala.reflect` is somewhat unstable.

Reply:
Do I really need it to? It will require messing up with IO Streams and it's so inelegant. I promise I fix it if it changes!